packages/teleport/src/services/audit/makeEvent.ts

```diff
@@ -889,6 +889,36 @@ export const formatters: Formatters = {
     format: ({ user, db_service, db_name, proc_name }) =>
       `User [${user}] has send RPC Request [${proc_name}] in database [${db_name}] on [${db_service}]`,
   },
+  [eventCodes.ELASTICSEARCH_REQUEST]: {
+    type: 'db.session.elasticsearch.request""',
+    desc: 'Elasticsearch Request',
+    format: ({ user, db_service, db_name, category, target, query, path }) => {
```

Comment:
`db_name` seems to be unused: 
```suggestion
    format: ({ user, db_service, category, target, query, path }) => {
```

Reply:
Done.